auth-web/src/stores/business.ts

```diff
@@ -323,14 +323,15 @@ export const useBusinessStore = defineStore('business', () => {
       case FilingTypes.REGISTRATION: {
         filingBody = {
           filing: {
+            business: {
+              legalType: business.nameRequest.legalType,
+              natureOfBusiness: business.nameRequest.natureOfBusiness
+            },
```

Comment:
The draft registration filing should have 2 business objects.

One at the top level, here, containing just legalType.

And the other under the registration object as in the previous code, containing just the NOB.